src/components/Calculator.js

```diff
@@ -1,10 +1,55 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
 import React from 'react';
-import CalculatorDesign from './CalculatorDesign';
 
-export default function Calculator() {
+function Calculator(props) {
+  const { calcState, handleClick } = props;
   return (
-    <div className="calculator">
-      <CalculatorDesign />
-    </div>
+    <>
+      <div className="display">
+        {calcState.result}
+        {calcState.function}
+        {calcState.next}
+      </div>
+      <div className="btnsContainer">
+        <div>
+          <button type="button" className="Btns" onClick={handleClick}>AC</button>
+          <button type="button" className="Btns" onClick={handleClick}>+/-</button>
+          <button type="button" className="Btns" onClick={handleClick}>%</button>
+          <button type="button" className="Btns MathOperations">÷</button>
```

Comment:
- [ ] I notice that you did not implement the click events on `÷` so it is not working please implement it and ask for another reviewer.
` <button type="button" className="Btns MathOperations" onClick={handleClick}>÷</button>`